CONTRIBUTING.rst

```diff
@@ -0,0 +1,480 @@
+You can access the documentation on the `DrizzlePac
+readthedocs
+website <https://readthedocs.org/projects/drizzlepac-pipeline/>`__ - the
+'latest' version is whatever is currently on the main branch. If you successfully merge
+a PR with documentation changes, they will only appear on 'latest' until
+the next DrizzlePac release.
+
+All of the documentation resides in the ``drizzlepac/docs``
+subdirectories (mainly within directories in
+``drizzlepac/docs/drizzlepac``, organized by module). The documentation
+is written in ``.rst`` (reStructured text) files - if you wish to make
+changes or add to the documentation, do so in these files in the
+appropriate location. reStructured text is the markup language used by
+sphinx, for information on the syntax refer to the `sphinx
+documentation <https://www.sphinx-doc.org/en/main/usage/restructuredtext/basics.html>`__.
+
+While writing documentation, you will want to make sure the
+documentation builds successfully (i.e, produces a working .html file).
+This happens on the CI when you open a pull request, but it is a good
+idea to build the documentation yourself locally as well. Before you do
+this, you will need to make sure you have the correct dependencies
+installed in your current environment. All of these optional
+dependencies are specified in ``pyproject.toml`` and include things like
+the correct version of sphinx, as well as the necessary sphinx themes
+that the project uses. These do not install automatically when you
+install ``drizzlepac`` unless directly specified. To do this, while in
+the top level directory of ``drizzlepac`` on your my_feature branch:
+
+::
+
+   >> pip install -e ".[docs]"
+
+Now, with the correct documentation dependencies installed, you can
+attempt to build the documentation locally. To do this, enter into the
+``drizzlepac/docs`` subdirectory and do:
+
+::
+
+   >> make html
+
+If the documentation successfully builds, the output HTML files will be
+output in the ``_build/html`` subdirectory. You can open the main
+``index.html`` file in your browser and explore the full documentation
+pages just like the readthedocs site. If there were any errors or
+warnings, a traceback will be printed on your terminal. Small errors,
+like a typo in a link to another section, can cause frustrating errors
+so it is good practice to build the docs frequently when editing them.
+
+Writing and running unit tests
+------------------------------
+
+Unit tests are located in the ``tests`` directory and are separated by instrument,
+with additional separate directories for Hubble Advanced Products (HAP) and the 
+drizzle algorithm. These tests run the code
+on simplified datasets to make sure there are no breaking changes
+introduced. We aim to cover most lines of ``drizzlepac`` with unit
+test, so when adding code you will often need to write a new test or add
+to an existing test to ensure adequate coverage.
+
+Take a look around at the existing tests for a template - a majority of
```

Comment:
I would re-word 348-354 as this is not really true of the drizzlepac tests.  A "datamodel" is not necessarily used and the simplified data/dq is also not usually true.  The HAP tests are real "as is" datasets.